manifest.yml

```diff
@@ -0,0 +1,8 @@
+applications:
+- name: response-operations-ui
+  buildpack: https://github.com/cloudfoundry/staticfile-buildpack.git
```

Comment:
No buildpack required for this service, it is only used in the backstage-ui (Angular app).